gattc.go

```diff
@@ -0,0 +1,5 @@
+package bluetooth
+
+import "errors"
+
+var ErrCannotSendWriteWithoutResponse = errors.New("cannot send write without response")
```

Comment:
It seems all other implementations already have its error handling (e.g. gattc_hci.go), so we should move this variable to "gattc_darwin.go". We need it exposed to use it in user code for checks, like you already demonstrated in your small example. (IMO unexposed variants in the other implementations are not that powerful, but this is another story.)